geode-core/src/main/java/org/apache/geode/distributed/internal/InternalDistributedSystem.java

```diff
@@ -751,12 +771,12 @@ private void initialize(SecurityManager securityManager, PostProcessor postProce
 
       if (!isLoner) {
         try {
-          dm = ClusterDistributionManager.create(this);
+          dm = ClusterDistributionManager.create(this, membershipLocator);
           // fix bug #46324
```

Comment:
completely aside... is this a TODO for an old bug or what follows was the fix for it? anyone, bueller...

Reply:
my impression when I see a comment like that is that it denotes a fix for that bug—not a TODO

Reply:
On your other comment, re: when would there not be a membershipLocator specified (to `MembershipBuilderImpl`?)

I'd say when building membership in a JVM that is not a locator—it's just a member.